src/r3/atomic-swap/corda/evm-bridge-workflows/src/main/kotlin/com/r3/corda/evmbridge/services/Connection.kt

```diff
@@ -303,7 +304,11 @@ internal class Connection(private val connectionId: ConnectionId) {
             block.gasLimit,
             block.gasUsed,
             block.timestamp,
-            block.transactions.map { mapTransaction(it as EthBlock.TransactionObject) },
+            block.transactions.map {
+                // TODO: when fullTransactionObject = false, the TransactionResult is the hash (String) of the
```

Comment:
Don't love leaving ToDo's in code we're going to merge but if the next PR you mentioned cleans this up its fine.